task-sdk/tests/task_sdk/execution_time/test_supervisor.py

```diff
@@ -1406,6 +1452,7 @@ def watched_subprocess(self, mocker):
                 (TI_ID, 1),
                 {},
                 TaskRescheduleStartDate(start_date=timezone.parse("2024-10-31T12:00:00Z")),
+                [],
```

Comment:
Nit: Maybe all these should be none instead?

Reply:
Yeah not an issue at all.